tests-bundle/1.7/test_tutorial.py

```diff
@@ -11,7 +11,7 @@
 
 TESTS_DIR = Path(__file__).resolve().parent
 
-
+@pytest.mark.usefixtures("failed_check")
```

Comment:
have you tested it locally ?

Reply:
Not yet. Will do

Reply:
@beliaev-maksim Yes I tested locally and the screenshots are indeed being generated. I added some files to the .gitignore in case someone is running these tests locally.